src/main/java/com/solvd/airport/AirportApplication.java

```diff
@@ -1,37 +1,51 @@
 package com.solvd.airport;
 
+import com.solvd.airport.Exceptions.InvalidDateException;
 import com.solvd.airport.classes.entity.*;
+import com.solvd.airport.classes.service.Booking;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
 public class AirportApplication {
 	private static final Logger logger = LogManager.getLogger(AirportApplication.class);
 
-	public static void main(String[] args) {
+	public static void main(String[] args) throws InvalidDateException {
```

Comment:
You should handle this Exception somewhere, not throw it out from main functionc